packages/node-postgres/test/integration/dsql.integration.test.ts

```diff
@@ -0,0 +1,134 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { jest, describe, test, expect } from "@jest/globals";
+import { AuroraDSQLClient } from "../../src/aurora-dsql-client";
+import { AuroraDSQLPool } from "../../src/aurora-dsql-pool";
+
+jest.setTimeout(30000);
+
+async function verifySuccessfulConnection(client: AuroraDSQLClient) {
+  try {
+    await client.connect();
+    const result = await client.query("SELECT 1 as test_value");
+    expect(result.rows[0].test_value).toBe(1);
+  } finally {
+    await client.end();
+  }
+}
+
+describe("DSQL Integration Tests", () => {
+  const clusterEndpoint = process.env.CLUSTER_ENDPOINT;
+  const region = process.env.REGION;
+
+  describe("AuroraDSQLClient", () => {
+    test("should connect to DSQL cluster", async () => {
+      const client = new AuroraDSQLClient({
+        host: clusterEndpoint,
+        database: "postgres",
+        user: "admin",
+        region: region,
+        port: 5432,
+      });
+      await verifySuccessfulConnection(client);
+    });
+
+    test("should connect without providing region", async () => {
+      const client = new AuroraDSQLClient({
+        host: clusterEndpoint,
+        database: "postgres",
+        user: "admin",
+        port: 5432,
+      });
+      await verifySuccessfulConnection(client);
+    });
+
+    test("should connect with minimum parameters", async () => {
+      const client = new AuroraDSQLClient({
+        host: clusterEndpoint,
+        user: "admin",
+      });
+      await verifySuccessfulConnection(client);
+    });
+
+    test("should handle connection string format", async () => {
```

Comment:
Have we tested what happens if you provide both a connection string and properties?